src/components/Progressbar/BProgressBar.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "react-step-progress-bar/styles.css";
+import { ProgressBar } from "react-step-progress-bar";
+import Grid from '@material-ui/core/Grid';
+
+function BProgressBar(){
+  return(
+    <div>
+      <Grid container 
+      spacing = {1}
+      justify ="center" 
+      alignItems = "center"
+      >
+      <Grid
+        item align = "center"
+        item xs ={5} sm={5} md={5} lg={5}>
+        Bias Score
```

Comment:
Can we talk about calling it a Bias Score? I have some reservations that this is the right word to describe your bar. Maybe 'Trust' or 'Trustworthiness'?

Reply:
Sure, we can talk through this. No change to the way the score is calculated. Refactoring would involve changing backend logic, calculations  as well and so would take it through the team discussion

Reply:
Oh the calculations are fine, no need to change anything there. I just think we need to think carefully about the title.